stress-example/nlp_search/flows/index.yml

```
@@ -0,0 +1,27 @@
+!Flow
+version: '1'
+pods:
+  - name: segmenter
+    show_exc_info: true
+    uses: pods/segment.yml
+    read_only: true
+  - name: encoder
+    show_exc_info: true
+    uses: pods/encoder.yml
+    parallel: $JINA_SHARDS_ENCODER
```

```suggestion
    shards: $JINA_SHARDS_ENCODER
```

Make sure you've replaced all usage of `parallel` with `shards`. It's the same, but it's a bit confusing to use both (we should just stick to one for consistency)